Makefile

```diff
@@ -8,6 +8,7 @@ VERSION := $(shell ./scripts/build/get-version.sh)
 BUILD_COMMIT := $(shell ./scripts/build/get-build-commit.sh)
 BUILD_TIMESTAMP := $(shell ./scripts/build/get-build-timestamp.sh)
 BUILD_HOSTNAME := $(shell ./scripts/build/get-build-hostname.sh)
+GOFMT_OUTPUT := $(shell gofmt -l . | grep -v ^vendor)
```

Comment:
why is this in a variable here?  it is used only in one target, and I can't imagine a scenario where we would want to use the feature it adds, of being able to override this variable from outside

Reply:
Removed please check